Orange/misc/tests/test_distmatrix_xlsx.py

```diff
@@ -115,6 +115,12 @@ def test_active_worksheet(self):
             self.assertTrue({"E15", "sheet", "empty"} & set(str(exc).split()))
 
 
+def test_write_nan_values():
```

Comment:
Please remove this function. It doesn't do anything (because it is not within test class) and can't work.